onnxruntime.spec

```diff
@@ -1,11 +1,11 @@
-### RPM external onnxruntime 0.5.0
-%define tag 2824909ae569932d9aee1462049ff0da1e766989
-%define branch cms/master/9f633c5b
+### RPM external onnxruntime 1.0.0
+%define tag dfec41d1371a7b1233d25db55561ce4021fc7ceb
+%define branch cms/v1.0.0
 %define github_user cms-externals
 Source: git+https://github.com/%{github_user}/%{n}.git?obj=%{branch}/%{tag}&export=%{n}-%{realversion}&submodules=1&output=/%{n}-%{realversion}.tgz
 
 BuildRequires: cmake ninja zlib python3
-Requires: eigen protobuf
+Requires: protobuf
```

Comment:
@hqucms , yes it is not needed as onnxruntime uses eigen internally i.e. non of the onnxruntime public headers include eigen. So removing it is correct. We need to be careful when onnxruntime public headers starts using eigen. In that case we have to make sure that onnxruntime and cmssw use the same eigen version.

Reply:
@smuzaffar Thank you very much for confirming that!